### PR TITLE
feat: add notification that the student already has a mentor

### PR DESCRIPTION
**Pull request by idzeravianka**
#### 🧑‍⚖️ Pull Request Naming Convention

 - Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
 - Do not put issue id in title
 - Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
 - Consider to add `area:*` label(s)

 * [x] I followed naming convention rules

---

#### 🤔 This is a ...
- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link
 #1056 

#### 💡 Background and solution 
implement notification that student already has a mentor

Screenshot 2021-11-18 at 11 03 19

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: server/src/repositories/student.repository.ts [hunk 1]
```diff
@@ -85,6 +88,7 @@ export class StudentRepository extends AbstractRepository<Student> {
       id: entity.id,
       githubId: entity.githubId,
       name: userService.createName(entity),
+      mentorGithub: entity.mentorGithub,
```
Comment (apalchys) [inline server/src/repositories/student.repository.ts:91]: please rename here and in other places to `mentorGithubId` for consistency (we used in some places already)

**Review by apalchys — CHANGES_REQUESTED** [review]
see above

File: server/src/repositories/student.repository.ts (hunk 1 above)
Comment (idzeravianka) [reply server/src/repositories/student.repository.ts]: done

**Comment by apalchys** [comment]
@idzeravianka Sorry, I made some changes. After careful review, I decided to make model more generic

**Review by apalchys — APPROVED** [review]

**Pull request merged**